<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chess Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fefefe;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QFrame" name="chessBoardFrame">
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>650</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"></string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightPanelLayout">
      <item>
       <widget class="QLabel" name="playerInfoLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Ход: Белые</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #1e3a5f; background-color: #dbeafe; padding: 12px; border-radius: 15px; border: 1px solid #93c5fd;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="currentMoveEdit">
        <property name="placeholderText">
         <string>e2 e4</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #374151; background-color: white; border: 1px solid #d1d5db; padding: 8px; border-radius: 8px; font-family: monospace; font-size: 14px;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacerTop">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="surrenderButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>Сдаться</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { color: white; background-color: #ef4444; border: 1px solid #dc2626; border-radius: 12px; padding: 10px; font-weight: bold; font-size: 12px; } QPushButton:hover { background-color: #f87171; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drawButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>Ничья</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { color: white; background-color: #fbbf24; border: 1px solid #f59e0b; border-radius: 12px; padding: 10px; font-weight: bold; font-size: 12px; } QPushButton:hover { background-color: #fcd34d; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="menuButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>Меню</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { color: white; background-color: #60a5fa; border: 1px solid #3b82f6; border-radius: 12px; padding: 10px; font-weight: bold; font-size: 12px; } QPushButton:hover { background-color: #93c5fd; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="historyButton">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>38</height>
         </size>
        </property>
        <property name="text">
         <string>История</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { color: white; background-color: #34d399; border: 1px solid #10b981; border-radius: 12px; padding: 10px; font-weight: bold; font-size: 12px; } QPushButton:hover { background-color: #6ee7b7; }</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacerButtons">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="verticalSpacerTimer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="timerLabel">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>00:30</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #059669; padding: 12px; background-color: #ecfdf5; border-radius: 10px; border: 1px solid #a7f3d0;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
